task/03-date-tasks.js

```diff
@@ -56,7 +56,11 @@ function parseDataFromIso8601(value) {
  *    Date(2015,1,1)    => false
  */
 function isLeapYear(date) {
-   throw new Error('Not implemented');
+   if (date.getUTCFullYear()%4 === 0){
+      return !(date.getUTCFullYear() % 400 !== 0 && date.getUTCFullYear() % 100 === 0); // if year is divisible by 4 and by 400, then it is a leap year
+   }
+   else
+      return false; // the year is not divisible by 4 then it is not a leap year
```

Comment:
All expressions in the function are logical expressions and they return the logic `true/false` result.